### ISSUE TITLE
IslandChat "$" character = IndexOutOfBoundsException

### ISSUE THREAD

**Issue by BlackBeltPanda** [rlf/uSkyBlock#810]
_Please paste the output from `/usb version` below_
```
Name: uSkyBlock
Version: 2.6.12
Description: Ultimate SkyBlock v2.6.12-9e2d30-413
Language: en (en)
------------------------------
Server: Paper git-Paper-806 (MC: 1.10.2)
------------------------------
Vault 1.5.6-b49 (ENABLED)
WorldEdit 6.1.3;7a097ca (ENABLED)
WorldGuard 6.1.2;e38d98d (ENABLED)
AsyncWorldEdit 3.3.12 (ENABLED)
Multiverse-Core 2.5-b719 (ENABLED)
Multiverse-NetherPortals 2.5-b710 (ENABLED)
------------------------------
```

_What steps will reproduce the problem?_
1. Execute "/it $"

_If you have any log-files, please paste them to [pastebin.com](http://pastebin.com)_
* server-log: http://pastebin.com/raw/NgkZTJe9

**Comment by BlackBeltPanda** [comment]
Escaping the "$" with "\$", such as "/it \$430", seems to work.

**Comment by rlf** [comment]
Caused by the use of `String.replaceAll()` - most likely present a lot of other places in the code.

**Comment by dutchy1001** [comment]
this issue is still present in my internal dev build (Ultimate SkyBlock v2.7.0-alpha1-9-gbf41952-bf4195-dev)

**Comment by dutchy1001** [comment]
fix in the latest (v2.7) release.
